qiskit_machine_learning/algorithms/classifiers/pegasos_qsvc.py

```diff
@@ -0,0 +1,285 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Pegasos Quantum Support Vector Classifier."""
+
+import logging
+from datetime import datetime
+from typing import Optional, Dict, Tuple
+
+import numpy as np
+from sklearn.svm import SVC
+
+from qiskit_machine_learning.exceptions import QiskitMachineLearningError
+from qiskit_machine_learning.kernels.quantum_kernel import QuantumKernel
+
+logger = logging.getLogger(__name__)
+
+
+class PegasosQSVC(SVC):
+    """
+    This class implements Pegasos Quantum Support Vector Classifier algorithm developed in [1]
+    and includes overridden methods ``fit`` and ``predict`` from the ``SVC`` super-class.
+
+    **Example**
+
+    .. code-block::
+        quantum_kernel = QuantumKernel()
+
+        pegasos_qsvc = PegasosQSVC(quantum_kernel=quantum_kernel)
+        pegasos_qsvc.fit(sample_train, label_train)
+        pegasos_qsvc.predict(sample_test)
+
+    **References**
+        [1]: Shalev-Shwartz et al., Pegasos: Primal Estimated sub-GrAdient SOlver for SVM.
+            `Pegasos for SVM <https://home.ttic.edu/~nati/Publications/PegasosMPB.pdf>`_
+
+    """
+
+    def __init__(
+        self,
+        quantum_kernel: Optional[QuantumKernel] = None,
+        C: float = 1,
```

Comment:
Is this the only parameter from the parent SVC we want to expose. In the SVC we have kwargs here that allows one to pass any of the parameters that the SVC supports without listing them explicitly here. And minor comment since its a float it might be nice to put it as `1.0` rather than `1`

Reply:
Replaced with `1.0`.
As I see from the implementation, this is what can be re-used right now.